src/mame/barcrest/mpu4.cpp

```diff
@@ -1958,16 +1851,17 @@ INPUT_PORTS_START( grtecp )
 INPUT_PORTS_END
 
 
-
 /* Common configurations */
 
 void mpu4_state::mpu4_install_mod4oki_space(address_space &space)
 {
-	pia6821_device *const pia_ic4ss = subdevice<pia6821_device>("pia_ic4ss");
+	mpu4_oki_sampled_sound *const m_soundcard = subdevice<mpu4_oki_sampled_sound>("okicard");
```

Comment:
Do you not have a device finder member for this thing?  Seems you already have `m_okicard`.  Also, you shouldn’t use the `m_` prefix for local variables – it will lead to confusion as it’s used to denote data members.